### DISCUSSION TITLE
Best way to Refresh Plot file for farming while Plotting

### DISCUSSION THREAD

**Discussion by bryantcj89** [madMAx43v3r/chia-plotter#790] [General]
I am using mad max for plotting NFT, may i know any best way to auto refresh the plot file instead of we manually go GUI to refresh it so that it can use the latest plot file for farming?

**Comment by flojones001** [comment]
if you pay attention to "plot past filter" in "farm", you 'll see that new plots are automatically taken into account within 1 or 2 minutes for farming. So even if "Plot count" is not updated, we don't care because that's not this number that counts for farming.
When pool farming, it is the number of proof found that wins you points. So we also don't care if "number of plots"  in "pool" is not updated automatically.
